src/GraphControl/Control/Grapher.cpp

```diff
@@ -699,3 +726,177 @@ namespace GraphControl
         return outputStream;
     }
 }
+
+void Grapher::OnCoreKeyUp(CoreWindow ^ sender, KeyEventArgs ^ e)
+{
+    // We don't want to react to keyboard input unless the graph control has the focus.
+    // NOTE: you can't select the graph control from the mouse for focus but you can tab to it.
+    FrameworkElement ^ whoHasFocus = (FrameworkElement ^) FocusManager::GetFocusedElement();
+    String ^ wName = whoHasFocus->Name;
+    String ^ ETBName = L"GraphingControl";
```

Comment:
We shouldn't hard code this, we should use this->Name instead. However, the GraphControl doesn't require a name either, people could use it without assigning a name. You can just check the self contained FocusState of the GraphControl or compare the returned FrameWorkElement to the GraphControl itself.

Reply:
Changed to us a dynamic_cast like c# "as" functionality.  If we get a nullptr back it isn't the graph control and we skip processing.

Reply:
Nice solution, I think we should still check that the GraphControl is really us in case there are multiple. Super NIT corner case :P